src/assets/i18n/english.json

```diff
@@ -13,7 +13,8 @@
     "add": "Add Bookmark",
     "external": "External",
     "name": "Bookmark",
-    "url": "Url"
+    "url": "Url",
+    "add-file": "Add File"
```

Comment:
These should be sorted alphabetically (otherwise some other things in the project will start acting up, and even forcefully change/fix these)

Reply:
Oh i didn't know that. Will update it.

Reply:
should be alphabatised